akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteNodeShutdownAndComesBackSpec.scala

```diff
@@ -118,8 +118,8 @@ abstract class RemoteNodeShutdownAndComesBackSpec
         watch(subjectNew)
 
         subjectNew ! "shutdown"
-        fishForMessage(5.seconds) {
-          case _: ActorIdentity       ⇒ false
+        receiveWhile(5.seconds) {
+          case _: ActorIdentity       ⇒ true
```

Comment:
shouldn't this be false? I.e. we keep pulling the stray ActorIdentity responses from previous test steps until we find the Terminated?